tests/integration/test_global.py

```diff
@@ -161,10 +161,18 @@ def test_sync_migrate(pixi: Path, tmp_path: Path, dummy_channel_1: str) -> None:
     manifests.mkdir()
     manifest = manifests.joinpath("pixi-global.toml")
     toml = f"""\
+# Test with special channel
 [envs.test]
 channels = ["{dummy_channel_1}"]
 dependencies = {{ dummy-a = "*", dummy-b = "*" }}
 exposed = {{ dummy-1 = "dummy-a", dummy-2 = "dummy-a", dummy-3 = "dummy-b", dummy-4 = "dummy-b" }}
+
+# Test with conda-forge channel
+[envs.test2]
+channels = ["conda-forge"]
```

Comment:
Would it make sense to add a `dummy-channel` here to make sure that everything also works with multiple channels within one environment?

Reply:
Thanks for doing it! @Hofer-Julian